WinUIGallery/Package.WAP.Debug.appxmanifest

```diff
@@ -0,0 +1,55 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package
+    xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+    xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+    xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+    xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
+    xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+    IgnorableNamespaces="uap mp uap3">
+  <Identity Name="Microsoft.WinUI3ControlsGallery.Debug" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="1.3.13.0" />
```

Comment:
Suggestion: use Dev instead of Debug in both the name and filename.  Following the Terminal model, we can introduce branding (not necessarily in this PR) so that "WinUI 3 Gallery" is always Release, "WinUI 3 Gallery Dev" is the default and can be Debug or Release

Reply:
I see what you are going after here. So would it be better to rename this to dev and make it the default for both debug and release and keep the "non-dev" version up just for building the store version?

Reply:
exactly :)